### ISSUE TITLE
Add support for Storybook 5

### ISSUE THREAD

**Issue by jamesmoss** [gvaldambrini/storybook-router#32]
Right now when installing this addon with storybook 5 you get this warning in yarn:

```
warning " > storybook-react-router@1.0.2" has incorrect peer dependency "@storybook/addon-actions@^4.0.0".
warning " > storybook-react-router@1.0.2" has unmet peer dependency "@storybook/addons@^4.0.0".
```

It seems like the addon works fine in SB5 so should be fine to bump the dependency to `^4.0.0||^5.0.0`.

**Comment by gvaldambrini** [comment]
Thank you for the report, I'll have a look asap.

**Comment by gvaldambrini** [comment]
Released version 1.0.3, with support for Storybook 5 😄

**Comment by jamesmoss** [comment]
Just upgraded thanks, one thing though - do you actually need to specify `@storybook/addons` in `dependencies` and `peerDependencies` at all? `@storybook/addons-actions` includes `@storybook/addons` as a dependency, and you don't reference it directly in the code in this package also. 

Right now I have to add `@storybook/addons` as a dependency for no reason to my `package.json` because you've specified it as a peer dependency.

**Comment by gvaldambrini** [comment]
You're right, fixed with 1.0.5. Thank you for the report.
